pkgs/top-level/all-packages.nix

```
@@ -25471,6 +25471,8 @@ with pkgs;
 
   dupd = callPackage ../tools/misc/dupd { };
 
+  jdownloader2 = callPackage ../applications/networking/jdownloader2 {};
```

Nit: one more space here will be nice.

```suggestion
  jdownloader2 = callPackage ../applications/networking/jdownloader2 { };
```